source/index.html

```diff
@@ -191,15 +190,15 @@
 				align-items: center;
 			}
 		</style>
-		<div id='nav-intro-background' class='background'>
+		    <div id='nav-intro-background' class='background'>
```

Comment:
```suggestion
		<div id='nav-intro-background' class='background'>
```

Reply:
I will just remove the unused fonts from the folder for now.